alf/environments/wrappers.py

```diff
@@ -53,18 +72,40 @@ def __init__(self, env, stack_size=4, channel_order='channels_last'):
             shape = [
                 stack_size * space.shape[0],
             ] + list(space.shape[1:])
-        self.observation_space = gym.spaces.Box(
+        box_space = gym.spaces.Box(
```

Comment:
should use the 'high' and 'low' from the original space

Reply:
changed as recommended.